src/index.new.test.ts

```diff
@@ -124,6 +125,43 @@ describe('Core Index Tests', () => {
     });
   });
 
+  describe('listIntegrationInstances', () => {
+    const setup = (): void => {
+      j1.graphClient.query = jest.fn().mockImplementation(() => {
+        return Promise.resolve({
+          data: {
+            integrationInstances: {
+              instances: [exampleIntegrationInstance],
+            },
+          },
+        });
+      });
+    };
+
+    beforeEach(() => {
+      setup();
+    });
+
+    test('Sad Test - Query Fails', async () => {
+      j1.graphClient.query = jest.fn().mockImplementation(() => {
+        return {
+          errors: [{ message: 'A Problem' }],
+        };
+      });
+
+      const expectedValue = { data: [], errors: [{ message: 'A Problem' }] };
+      const test = await j1.integrationInstances.list();
+      await expect(test).toEqual(expectedValue);
+    });
+
+    test('Happy Test - Returns Instances', async () => {
```

Comment:
I see that you were following an existing convention but wow these test names are weirdo. 😆